src/targets/elixir/httpoison.js

```diff
@@ -0,0 +1,134 @@
+/**
+ * @description
+ * HTTPoison Elixir HTTP client code snippet generator.
+ *
+ * @author
+ * @danirukun
+ *
+ * For any questions or issues regarding the generated code snippet,
+ * please open an issue mentioning the author.
+ */
+
+'use strict'
+
+const CodeBuilder = require('../../helpers/code-builder')
+
+/**
+   Converts an ES6 list of objects to an Elixir KW list
+*/
+const jsListToKwList = (jsList) => {
+  if (jsList.length === 0) {
+    return '[]'
+  } else {
```

Comment:
nit: unnecessary `else` given the `return`

Reply:
Hmm what do you mean by this?

Reply:
[This](https://github.com/Kong/httpsnippet/pull/235/files#r779199855) should be functionally equivalent

Reply:
Refactored, thank you for the explanation!